### DISCUSSION TITLE
Get-PnPSiteSearchQueryResults : Value cannot be null.

### DISCUSSION THREAD

**Discussion by IfEyeOnlyKnew** [pnp/powershell#2130] [Q&A]
No issues running Get-PnPTenantRecycleBinItem and getting results.

Anyone seeing this issue using the PnP Command Get-PnPSiteSearchQueryResults this morning 7/11/2022? Just confirming before putting in an Issue. :/

`$tenantadminurl = "https://tenanat-admin.sharepoint.com"

Connect-PnPOnline -url $tenantadminurl -Interactive

Get-PnPSiteSearchQueryResults

Get-PnPSiteSearchQueryResults -MaxResults 10

Get-PnPSiteSearchQueryResults - All
`

Get-PnPSiteSearchQueryResults : Value cannot be null.
Parameter name: context
At line:1 char:1
+ Get-PnPSiteSearchQueryResults -MaxResults 10
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : WriteError: (:) [Get-PnPSiteSearchQueryResults], ArgumentNullException
    + FullyQualifiedErrorId : EXCEPTION,PnP.PowerShell.Commands.Search.GetSiteSearchQueryResults

**Comment by IfEyeOnlyKnew** [comment]
This issue seems to be related to our tenant. I have a MS Support ticket opened.

**Reply by fowl2** [reply]
Looking at the code, I'm not sure how it ever worked. It certainly doesn't for me.

For reference, on PowerShell 7 the error is a little different:
 `Get-PnPSiteSearchQueryResults: Value cannot be null. (Parameter 'context')`

I've proposed a fix in #2138.
